src/m365/planner/commands/task/task-add.ts

```diff
@@ -279,7 +301,8 @@ class PlannerTaskAddCommand extends GraphCommand {
       { option: '--assignedToUserIds [assignedToUserIds]' },
       { option: '--assignedToUserNames [assignedToUserNames]' },
       { option: '--description [description]' },
-      { option: '--orderHint [orderHint]' }
+      { option: '--orderHint [orderHint]' },
+      { option: '--priority [priority]', autocomplete: ["Urgent", "Important", "Medium", "Low"] }
```

Comment:
I think it would be really sweet to add some kind of `taskPriority` class where we could add this list as static and just reuse it from that class here and in all other places. also in the `tast-set` command as well.
Similar to the [Outlook](https://github.com/pnp/cli-microsoft365/blob/main/src/m365/outlook/Outlook.ts) class
What do you think?